virttest/iscsi.py

```diff
@@ -807,9 +807,12 @@ class Iscsi(object):
     def create_iSCSI(params, root_dir=data_dir.get_tmp_dir()):
         iscsi_instance = None
         ubuntu = distro.detect().name == "Ubuntu"
+        suse = distro.detect().name == "SuSE"
         # check and install iscsi initiator packages
         if ubuntu:
             iscsi_package = ["open-iscsi"]
+        elif suse:
+            iscsi_package = ["open-iscsi"]
         else:
```

Comment:
Can we collapse these 2 cases into one single statement 

if ubuntu or suse
   iscsi_package = open-iscsi